src/ngx_lua_resty_lmdb_module.c

```diff
@@ -339,6 +415,113 @@ static ngx_int_t ngx_lua_resty_lmdb_init_worker(ngx_cycle_t *cycle)
 }
 
 
+static ngx_int_t
+ngx_lua_resty_lmdb_close_file(ngx_cycle_t *cycle,
+                              ngx_lua_resty_lmdb_conf_t *lcf)
+{
+    mdb_txn_abort(lcf->ro_txn);
+    mdb_env_close(lcf->env);
+
+    lcf->env = NULL;
+
+    return NGX_OK;
+}
+
+
+static ngx_int_t
+ngx_lua_resty_lmdb_verify_file_status(ngx_cycle_t *cycle,
+                                      ngx_lua_resty_lmdb_conf_t *lcf)
+{
+    ngx_core_conf_t  *ccf;
+    ngx_file_info_t   fi;
+
+    u_char            name_buf[NGX_MAX_PATH];
+    ngx_str_t        *name;
+    ngx_str_t         names[] = { ngx_string("/"),
+                                  ngx_string("/data.mdb"),
+                                  ngx_string("/lock.mdb"),
```

Comment:
These are defined multiple times, suggest we move it to module level instead.

Reply:
These two arrays have different purposes, one is for deleting, and the other is for permission.

So the first one has no `/`, but the second one has `/` to check the directory.

Reply:
defined a static array.